pkg/segment/aggregations/segaggs_test.go

```diff
@@ -2811,6 +2804,18 @@ func Test_NoWindow_StreamStats(t *testing.T) {
 	expectedFuncValues3 := [][]float64{expectedValuesCount, expectedValuesSum, expectedValuesSum, expectedValuesMax, expectedValuesMin, expectedValuesMin, expectedValuesCardinality}
 
 	NoWindowStreamStatsHelperTest(t, values, ssOption, measureFunctions, expectedFuncValues, expectedFuncValues2, expectedFuncValues3)
+
+	ssOption.Global = false
+
+	NoWindowStreamStatsHelperTest(t, values, ssOption, measureFunctions, expectedFuncValues, expectedFuncValues2, expectedFuncValues3)
+
+	ssOption.Current = false
+
+	NoWindowStreamStatsHelperTest(t, values, ssOption, measureFunctions, expectedFuncValues, expectedFuncValues2, expectedFuncValues3)
+
+	ssOption.Global = true
+
+	NoWindowStreamStatsHelperTest(t, values, ssOption, measureFunctions, expectedFuncValues, expectedFuncValues2, expectedFuncValues3)
```

Comment:
It looks like you're running this same function for all combinations of `Current` and `Global`. Would it make sense to have use loops here instead?

Same comment elsewhere

Reply:
updated in the latest commit.